RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Release notes
 
+## New in git-machete 3.2.2
+- improved: `show` can accept a target `--branch` other than the current branch (contributed by @asford)
+
 ## New in git-machete 3.2.1
 
```

Comment:
Whoops forgot to add `(contributed by @asford)` below, sorry 🙄

Reply:
I wouldn't risk editing these files tho, not sure if Debian changelogs won't get broken after such edit :/